static/js/dynamic-contact-form.js

```diff
@@ -200,6 +200,19 @@
       return message;
     }
 
+    // move g-recaptcha-response to marketo field
```

Comment:
Why have you duplicated this here? It seems like everywhere that you're linking to this JS file, you're then also calling `{% include "shared/forms/_recaptcha.html" %}` right after, which contains this same JS block. Unless I missed something?

Reply:
because this page is already a giant js function?  the include has script tags, etc...

Reply:
That doesn't make any sense to me. That block of code requires `grecaptcha` to be defined. This is added to the page by `<script src="https://www.google.com/recaptcha/api.js" async defer></script>`, which is only included in `templates/shared/forms/_recaptcha.html`. So you, quite rightfully, add a `<script>` block which uses `grecaptcha` right after that line, inside `templates/shared/forms/_recaptcha.html`.

So why do you also need it here? The block in the include should be sufficient.

Reply:
You include both in every case:

``` bash
 $ rg -A 2 dynamic-contact-form.min.js
templates/shared/forms/interactive/_general.html
257:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
258-{% include "shared/forms/_recaptcha.html" %}

templates/shared/forms/interactive/_support.html
227:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
228-{% include "shared/forms/_recaptcha.html" %}

templates/shared/forms/interactive/_bootstack.html
372:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
373-{% include "shared/forms/_recaptcha.html" %}

templates/shared/forms/interactive/_kubernetes.html
288:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
289-{% include "shared/forms/_recaptcha.html" %}

templates/shared/forms/interactive/_robotics.html
241:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
242-{% include "shared/forms/_recaptcha.html" %}

templates/shared/forms/interactive/_embedded.html
224:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
225-{% include "shared/forms/_recaptcha.html" %}

templates/shared/forms/interactive/_openstack.html
374:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
375-{% include "shared/forms/_recaptcha.html" %}

templates/shared/forms/interactive/_kubeflow.html
203:<script src="{% versioned_static 'js/build/dynamic-contact-form.min.js' %}"></script>
204-{% include "shared/forms/_recaptcha.html" %}
```